wiki/Shortcut_key_reference/en.md

```diff
@@ -71,7 +71,7 @@ These shortcuts work anywhere:
 | `Shift` + `F2` | Go back to the previously random-selected beatmap. |
 | `F3` | Open the beatmap options. |
 | `F5` | Reprocess beatmaps (this may prompt you to continue). |
-| `Shift` + `Delete` | Prompt the delete beatmap dialog. This will ask you if you want to delete the beatmapset (all difficulties), the beatmap (the selected difficulty), or all visible beatmaps. |
+| `Shift` + `Delete` | Prompt the delete beatmap dialog. This will ask you if you want to delete the beatmap (all difficulties), a single difficulty, or all visible difficulties. |
```

Comment:
```suggestion
| `Shift` + `Delete` | Prompt the delete beatmap dialog. This will ask you if you want to delete the beatmap (all difficulties), a single difficulty, or all visible beatmaps. |
```